jsx/Filter.js

```diff
@@ -75,6 +79,30 @@ class Filter extends Component {
     }, []);
   }
 
+  renderActions() {
+    if (this.props.actions) {
+      return this.props.actions.map((action) => {
+        if (action.show) {
+          // FIXME: This button prop logic will change once CTA buttons are no
+          // longer placed in the menu filter.
```

Comment:
Does the fact that #4203 (the refactoring to pass CTAs as a prop) is merged affect this?

Reply:
Yes, definitely. I think the fact that it's merged may cause some duplication of code within my PR. I can rebase and refactor my PR using the structure implemented by #4203